index.html

```diff
@@ -275,9 +275,48 @@ <h2 id="principal-title" class="color-blue fonts">About Myself
 
     <div class="divisor">
     </div>
+
+    <section id="form-section">
+      <div id="contact-container">
+        <div>
+          <h2 class="white">Contact me</h2>
+          <p class="white">If you have an application you are interested in developing, a feature that you need built or
+            a project that needs coding.
+            I’d love to help with it! </p>
+        </div>
+        <form class="form2" id="form" action="https://formspree.io/f/mqkoqdwg" method="POST">
+
+
+          <ul id="contactInGrid">
+            <li>
+              <div class="input-div">
+                <input type="text" name="name" maxlength="30" required="true" placeholder="Your Name" />
+              </div>
+            </li>
+            <li>
+              <div class="input-div">
+                <input id="mail" type="email" name="email" required="true" placeholder="Your email Address" />
+              </div>
+            </li>
+          </ul>
+          </li>
+
+          <div id="maxText" class="input-div">
+            <textarea name="message" required="true" id="message" maxlength="500"
+              placeholder="Write your message here..." rows="10"></textarea>
+          </div>
+          </li>
+
+
+        </form>
+        <button type="button" class="see-btn">Get in touch</button>
+      </div>
```

Comment:
- Kindly make sure that you get in touch button has a  `type` attribute note that it defaults to a `submit` and the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. :+1: